package.json

```diff
@@ -4,8 +4,9 @@
 	"description": "Spark bot that handles team recognition",
 	"main": "server.js",
```

Comment:
Doesn't this need to be changed to .ts?

Reply:
Not sure. It gets compiled into server.js and run as such